Background Scripts/Add Bookmarks - ITIL Users/script.js

```diff
@@ -0,0 +1,25 @@
+var jsonFavList = {
+  "SLA for My Group Tasks": "task_list.do?sysparm_query=assignment_groupDYNAMICd6435e965f510100a9ad2572f2b47744&sysparm_first_row=1&sysparm_view=",
+  "SLA for My Tasks": "task_list.do?sysparm_query=assigned_toDYNAMIC90d1921e5f510100a9ad2572f2b477fe&sysparm_first_row=1&sysparm_view=",
+  "Tasks Assigned to Me": "stateNOT INclosed_complete,closed_abandoned^assigned_toDYNAMIC90d1921e5f510100a9ad2572f2b477fe",
+  "My approvals": "sysapproval_approver_list.do?sysparm_query=approverDYNAMIC90d1921e5f510100a9ad2572f2b477fe&sysparm_first_row=1&sysparm_view="
```

Comment:
Hi @Jaspalsinghchot , is the URL for the **Tasks Assigned to Me** correct? Can you please check once. Same thing I mentioned in previous PR ( https://github.com/ServiceNowDevProgram/code-snippets/pull/1472 )

Reply:
@am-shakeel yes it is. Do you feel something incorrect?

Reply:
@am-shakeel updated.